Components/alert/alert.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pallet UI | Alert</title>
+
+    <!-- Favicon -->
+    <link rel="icon" href="/favicon.png">
+
+    <!-- font awesome link -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css"
+        integrity="sha512-PgQMlq+nqFLV4ylk1gwUOgm6CtIIXkKwaIHp/PAIWHzig/lKZSEGKEysh0TCVbHJXCLN7WetD8TFecIky75ZfQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- font family link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bitter:wght@600&display=swap" rel="stylesheet">
+
+    <!-- CSS links -->
+    <link rel="stylesheet" href="/Common CSS/style.css">
+    <link rel="stylesheet" href="/Components/alert/alert.css">
+    <link rel="stylesheet" href="/Common CSS/sidebar.css">
+    <link rel="stylesheet" href="/Common CSS/docs.css">
+
+</head>
+
+<body>
+
+    <div class="container">
+        <header class="nav">
+            <div class="nav-section">
+                <img src="/logo.svg" class="nav-logo" alt="">
+                <!-- <span class=" ui-version">v1.0</span> -->
+
+                <div class="nav--align-start">
+                    <a href="#" class="nav-links">Home</a>
+                    <a href="#" class="nav-links">Documentation</a>
+                </div>
+
+                <div class="nav--align-end">
+                    <i class="fa fa-moon" aria-hidden="true"></i>
```

Comment:
hey ! you can try adding title attribute to `i`element.

Reply:
Sure I will do that, thanks